### PR TITLE
Add Cranelift meeting agendas through May; add item on Jan 10.

### PR DESCRIPTION
**Pull request by cfallin**
This PR fills out meeting agenda templates for Cranelift through May 16. (May 30 is a US holiday so no meeting then.)

This PR also adds an item on Jan 10 to discuss #3612.

### CODE REVIEW

File: meetings/cranelift/2022/cranelift-02-07.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Febuary 7 project call
```
Comment (bnjbvr) [inline meetings/cranelift/2022/cranelift-02-07.md:1]: uber-nit

```suggestion
# February 7 project call
```

File: meetings/cranelift/2022/cranelift-01-10.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+**See the [instructions](../README.md) for details on how to attend**
+
+## Agenda
+1. Opening, welcome and roll call
+    1. Note: meeting notes linked in the invite.
+    1. Please help add your name to the meeting notes.
+    1. Please help take notes.
+    1. Thanks!
+1. Announcements
+    1. _Sumbit a PR to add your announcement here_
```
Comment (bnjbvr) [inline meetings/cranelift/2022/cranelift-01-10.md:12]: Don't know if this was a typo that became a recurring joke, but was always amused by this `Sumbit` here and below ;)

**Review by bnjbvr — APPROVED** [review]

File: meetings/cranelift/2022/cranelift-02-07.md (hunk 1 above)
Comment (cfallin) [reply meetings/cranelift/2022/cranelift-02-07.md]: Thanks!

File: meetings/cranelift/2022/cranelift-01-10.md (hunk 2 above)
Comment (cfallin) [reply meetings/cranelift/2022/cranelift-01-10.md]: Many eyeballs have passed over this but yours are the first to see the typo! Thanks!

**Pull request merged**